pkg/oomstore/push_processor.go

```diff
@@ -209,6 +210,6 @@ func (p *PushProcessor) newRevision(ctx context.Context, s *OomStore, groupID in
 	return nil
 }
 
-func (p *PushProcessor) lastRevision(unixMill int64) int64 {
-	return (unixMill / p.revisionInteval.Milliseconds()) * p.revisionInteval.Milliseconds()
+func (p *PushProcessor) lastRevision(revisionInterval int64, unixMill int64) int64 {
```

Comment:
snapshotInterval or revisionInterval

Reply:
Currently, I have written `revisionInterval`. Do you think `snapshotInterval` is better?
Mostly I'm not quite sure which one to use here, both make sense.

Reply:
Do you mean to modify `lastRevision` to `snapshotInterval` or `revisionInterval` ?

Reply:
I notice that line 146 uses `SnapshotInterval`, but here uses `revisionInterval`, would be better to consolidate

Reply:
both changed to `SnapshotInterval `